crates/sui-framework/packages/sui-framework/tests/crypto/zklogin_verified_id_tests.move

```diff
@@ -4,17 +4,16 @@
 #[test_only]
 module sui::zklogin_verified_id_tests {
     use sui::zklogin_verified_id::{check_zklogin_id, verify_zklogin_id};
-    use std::string::utf8;
     use sui::test_scenario;
 
     #[test]
     #[expected_failure(abort_code = sui::zklogin_verified_id::EFunctionDisabled)]
     fun test_check_zklogin_id() {
         let address = @0x1c6b623a2f2c91333df730c98d220f11484953b391a3818680f922c264cc0c6b;
-        let kc_name = utf8(b"sub");
-        let kc_value = utf8(b"106294049240999307923");
-        let aud = utf8(b"575519204237-msop9ep45u2uo98hapqmngv8d84qdc8k.apps.googleusercontent.com");
-        let iss = utf8(b"https://accounts.google.com");
+        let kc_name = b"sub".to_string();
+        let kc_value = b"106294049240999307923".to_string();
+        let aud = b"575519204237-msop9ep45u2uo98hapqmngv8d84qdc8k.apps.googleusercontent.com".to_string();
+        let iss = b"https://accounts.google.com".to_string();
```

Comment:
Just to make sure: `to_string` here is calling `utf8` correct?

Reply:
Yep!